quickstart/latest/Fiddler_Quickstart_Ranking_Model.ipynb

```diff
@@ -17,8 +17,15 @@
     "id": "b1yVGAH2vGvA"
    },
    "source": [
-    "# Quickstart: Expedia Search Ranking\n",
-    "The following dataset is coming from Expedia. It includes shopping and purchase data as well as information on price competitiveness. The data are organized around a set of “search result impressions”, or the ordered list of hotels that the user sees after they search for a hotel on the Expedia website. In addition to impressions from the existing algorithm, the data contain impressions where the hotels were randomly sorted, to avoid the position bias of the existing algorithm. The user response is provided as a click on a hotel. From: https://www.kaggle.com/c/expedia-personalized-sort/overview"
+    "### Example: Expedia Search Ranking\n",
+    "The following dataset is coming from Expedia. It includes shopping and purchase data as well as information on price competitiveness. The data are organized around a set of “search result impressions”, or the ordered list of hotels that the user sees after they search for a hotel on the Expedia website. In addition to impressions from the existing algorithm, the data contain impressions where the hotels were randomly sorted, to avoid the position bias of the existing algorithm. The user response is provided as a click on a hotel. From: https://www.kaggle.com/c/expedia-personalized-sort/overview\n",
```

Comment:
"The following dataset comes from Expedia" ???

Reply:
Reworded